src/components/HomeThumbnail.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import '../styles/home-thumbnail.css';
+import { getCoins } from '../redux/home/Home';
+
+export default function HomeThumbnail() {
+  const coins = useSelector((state) => state.coins);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getCoins());
+  }, [dispatch]);
+
+  return (
+    <div className="home-coin-container">
```

Comment:
- [ ] It would be better if we implement the filter functionality. Here we can filter by can, for example; 'ether`
![Screenshot from 2022-12-01 01-26-47](https://user-images.githubusercontent.com/46388113/204944450-0e757470-2363-4d5d-836c-5299babca5cd.png)